roles/openshift_hosted/tasks/router/router.yml

```diff
@@ -71,9 +71,6 @@
     - key: spec.strategy.rollingParams.updatePeriodSeconds
       value: 1
       action: put
-    - key: spec.strategy.activeDeadlineSeconds
-      value: 21600
-      action: put
```

Comment:
Removing spec.strategy.activeDeadlineSeconds causes this to report changed each time.